README.md

```diff
@@ -100,7 +101,7 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+it depends on ui-router and it is used a link in angular similar to link_to in rails.
```

Comment:
:+1: yes but it specifically directs us to a new state in our Angular app. `link_to` corresponds to using `href`